rust/stdin.rs

```diff
@@ -0,0 +1,15 @@
+// Test script to get input, transform, and write to stdout
+use std::io::{self, BufRead};
+
+fn main() {
+    let stdin = io::stdin();
+    let lines = stdin.lock().lines().map(|line| line.unwrap());
+
+    for (counter, line) in lines.enumerate() {
+        let line = line.trim();
```

Comment:
```suggestion
    let stdin = io::stdin();
    for (counter, line) in stdin.lock().lines().enumerate() {
        let line = line.unwrap().trim().to_string();
```

(just organized a bit differently)